app/main.py

```diff
@@ -16,4 +16,11 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [my_favourite_films,
+                                marks,
+                                collection_of_coins],
+                    "immutable": [lucky_number, pi,
+                                  one_is_a_prime_number,
+                                  name,
+                                  profile_info]
+                    }
```

Comment:
fix the dict brackets like this 
![image](https://user-images.githubusercontent.com/103201010/215498845-ff9cc0f0-94d7-4b9a-a8c7-d5191e57a5d3.png)